integrations/extensions/starter-kits/coveo/README.md

```diff
@@ -39,73 +41,74 @@ If you want to add this starter kit to an _existing_ assistant, you cannot use t
 
-- Click "New Step" and change "without conditions" to "with conditions" and select "Ran successfully" is "true" and "link2" is "defined".  For this step, set "And then" to "End the action".  Then add the following to the "Assistant says":
+- Click on the "Show search result" action and Add step 1:
+  - In "Variable values" set "link" to "$search_result.clickUri", set "title" to "$search_result.title", and set "snippet" to "$search_result.excerpt"
+  - Then add the following to the "Assistant says" to display the result and then set "And then" to "End the action".
 
-```
-<a href="${link2}" target="_blank">${title2}</a>
-${snippet2}
-```
+    ```
+    <a href="${link}" target="_blank">${title}</a>
+    ${snippet}
+    ```
 
-![Create third result snippet](./assets/define_response_3.png)<br>
+  - Close the "Show search result" action.
+
+  ![Show search result](./assets/show-search-result.png)<br>
 
 - Close the action editor (by clicking X in the upper right)
 - Go to "Actions" > "Set by assistant" > "No action matches" and remove all the steps from the action.  Add in a new step.  Under "And then" select "Go to another action" and select "Search" and click "End this action after the subaction is completed".
 - You may also want to go to "Actions" > "Set by assistant" > "Fallback" and do the same thing as in the previous step.  Note, however, that this will prevent your assistant from escalating to a human agent when a customer asks to connect to a human agent (which is part of the default behavior for "Fallback") so only do this if you do not have your bot connected to a human agent chat service.  For more details on connecting to human agents within Watson Assistant see [our documentation](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-human-agent) and [blog post](https://medium.com/ibm-watson/bring-your-own-service-desk-to-watson-assistant-b39bc920075c).
 - Go to the Search action and remove "Search" from the "Customer starts with" list so that the search action _only_ triggers via the "Go to another action" settings described in steps above.  If you skip this, then the "Search" action will also be considered by the intent recognizer as a possible intent, which adds unnecessary complexity to the intent recognition and thus could result in lower overall intent recognition accuracy.
 
 ![Create no action matches fallback](./assets/no_action_fallback_search.png)<br>
+
 ## Using this Starter Kit
 
 Once this starter kit is properly installed, you can issue a query to your bot and if there is no other action that you've configured that matched that query then it will generate search results for that query.
 
-Feel free to contribute to this starter kit, or add other starter kits by following these [contribution guidelines](../../docs/CONTRIBUTING.md).
\ No newline at end of file
+Feel free to contribute to this starter kit, or add other starter kits by following these [contribution guidelines](../../docs/CONTRIBUTING.md).
```

Comment:
The comma in this sentence should not be here because it is not a separate clause